tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -310,3 +310,148 @@ user enters text into meta guidance data file content editor
     user clicks element  ${editor}
     user waits until page contains testid   fileGuidanceContent-focused
     user enters text into element  ${editor}  ${text}
+
+
+user adds summary text block to content section
+    [Arguments]  ${text}
+    user clicks button  Add a summary text block 
+    user clicks button  Edit block 
+    user enters text into element  //*[@role="textbox"]//p  ${text} 
+    user clicks button  Save
+
+
+user adds release note to content 
+    [Arguments]  ${text}
+    user clicks button  Add note
+    user enters text into element  //*[@id="createReleaseNoteForm-reason"]  ${text}
+
+user creates amendment for release
+    [Arguments]  ${PUBLICATION_NAME}  ${RELEASE_NAME}  ${RELEASE_STATUS}
+    user goes to url  %{ADMIN_URL}
```

Comment:
I think you could use the `admin navigates to admin dashboard` keyword here? Although it might need adapting because at present it assumes you're logged in as Bau1.

Reply:
Yep that was the reason I used this instead.

Reply:
Refactored into a keyword: user navigates to admin dashboard as analyst1. Potentially might be worth while to refactor the user navigates to admin dashboard keyword to be more generic in nature